app/assets/javascripts/mobile.js

```diff
@@ -30,10 +31,18 @@ $(document).ready(function(){
 
   /* Drawer menu */
   $('#menu_badge').bind("tap click", function(evt){
```

Comment:
There is a lag here on mobile devices - more precisely 300ms; maybe we can use fastclick? The lag is introduced in order to see if the user didn't intend actually to double-tap. With fastclick, pinch-to-zoom would still work, not sure about double-tapping though.